workflows/dfe.snake

```diff
@@ -435,43 +383,51 @@ rule get_grapes_bestfits:
 
 rule plot_results:
     input:
-        dadi_res = expand(output_dir + "/inference/{demog}/dadi/{dfes}/{annots}/{seeds}/{chrms}/pop{ids}/pop{ids}.dadi.bestfit", 
+        dadi_res = expand(output_dir + "/inference/{demog}/dadi/{dfes}/{annots}/{seeds}/pop{ids}/pop{ids}.dadi.bestfit", 
             demog=demo_model_id_list,
             seeds=seed_list,
             dfes=dfe_list,
             annots=annotation_list,
-            chrms=chrm_list, 
-            ids=[0,1,2]
+            ids=pids,
         ),
-        polydfe_res = expand(output_dir + "/inference/{demog}/polyDFE/{dfes}/{annots}/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.bestfit",
+        polydfe_res = expand(output_dir + "/inference/{demog}/polyDFE/{dfes}/{annots}/{seeds}/pop{ids}/pop{ids}.polyDFE.bestfit",
             demog=demo_model_id_list,
             seeds=seed_list,
             dfes=dfe_list,
             annots=annotation_list,
-            chrms=chrm_list,
-            ids=[0,1,2]
+            ids=pids,
         ),
-        dfe_alpha_res = expand(output_dir + "/inference/{demog}/DFE-alpha/{dfes}/{annots}/{seeds}/{chrms}/pop{ids}/pop{ids}.DFE-alpha.bestfit",
+        dfe_alpha_res = expand(output_dir + "/inference/{demog}/DFE-alpha/{dfes}/{annots}/{seeds}/pop{ids}/pop{ids}.DFE-alpha.bestfit",
             demog=demo_model_id_list,
             seeds=seed_list,
             dfes=dfe_list,
             annots=annotation_list,
-            chrms=chrm_list,
-            ids=[0,1,2]
+            ids=pids,
         ),
-        grapes_res = expand(output_dir + "/inference/{demog}/grapes/{dfes}/{annots}/{seeds}/{chrms}/pop{ids}/pop{ids}.grapes.bestfit",
+        grapes_res = expand(output_dir + "/inference/{demog}/grapes/{dfes}/{annots}/{seeds}/pop{ids}/pop{ids}.grapes.bestfit",
             demog=demo_model_id_list,
             seeds=seed_list,
             dfes=dfe_list,
             annots=annotation_list,
-            chrms=chrm_list,
-            ids=[0,1,2]
+            ids=pids,
         )
     output:
-        output_dir + "/plots/{demog}/{chrms}/{dfes}/{annots}/dfe.inference.benchmark.pdf"
+        output_dir + "/plots/{demog}/{dfes}/{annots}/dfe.inference.benchmark.pdf"
     run:
-        contig = species.get_contig(wildcards.chrms)
-        seq_len = contig.recombination_map.sequence_length
+        # calc seq_len for dadi
```

Comment:
Whereas before we used the seq_len for each chromosome, here summing all the lengths to get genome length